src/main/webapp/app/overview/course-exercises/course-exercises.component.html

```diff
@@ -70,7 +69,7 @@ <h3 class="text-primary">
         <jhi-side-panel [panelHeader]="'artemisApp.courseOverview.exerciseList.details.courseInformation' | translate">
             <div class="row mb-1">
                 <div class="col-8">{{ 'artemisApp.courseOverview.exerciseList.details.totalExercises' | translate }}</div>
-                <div class="col-4">{{ course?.exercises.length }}</div>
+                <div class="col-4">{{ getNrOfExercises() }}</div>
```

Comment:
This can hurt performance as the function is called all the time. It would be better to assign the totalNumber of exercises to a component attribute.

Reply:
I created a variable